### DISCUSSION TITLE
マイページで初期表示されるタブに関して

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#532] [要望・アイデア・不具合]
# 概要

マイページで初期表示されるタブに関して、
現状は「投稿した記事」が初期表示されると思いますが、「投稿した記事」が無い場合は「投稿した質問」を表示するようにしてはどうか。（もしくは、「投稿した記事」件数と「投稿した質問」件数の多い方）

# 期待される効果

Q&Aする方々の多くは、Q&Aするために新たにアカウント開設されています。そういった方は記事の投稿がありません。
一方、Q&Aする方々の多くは、実質的に完了しているにも関わらず、オープンのまま放置される方が非常に多く居られます。
そこで、マイページの初期表示タブを変更すれば、「クローズ漏れ」に気付きを与えることができるのでは無いか、と考えました。
（そもそも、用済み後は「クローズする」という運用を知らない可能性もある）

関連チケット：https://github.com/increments/qiita-discussions/discussions/487

ご検討おねがしいます。

**Comment by tanaka-asami415** [comment]
@nak435 
ご提案ありがとうございます！🙏

頂いた内容をそのまま採用することは難しそうですが、
マイページを開いた際に、質問も確認してもらいやすい状態にできないかは検討したいと思います！
ただ、直近の開発優先度などからすぐの対応はできないため、ご了承いただけると幸いです🙇
